src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+| ShortChannelCount   | RECOMMENDED           | [number][]            | The number of short channels. 0 indicates no short channels.                                                                                                                                                                                                                                                                                                                          |
+| NIRSPlacementScheme | RECOMMENDED           | [string][]            | Placement scheme of NIRS optodes. Either the name of a standardized placement system (for example, "10-20") or a list of standardized position names (for example, ["Cz", "Pz"]). This field should only be used if a cap was not used. If a standard cap was used then it should be specified in CapManufacturer and CapManufacturersModelName and this field should be set to “n/a” |
+
+Example:
+
+```JSON
+{
+  "TaskName":"visual",
+  "InstitutionName":"Macquarie University. Australian Hearing Hub",
+  "InstitutionAddress":"6 University Ave, Macquarie University NSW 2109 Australia",
+  "Manufacturer":"NIRx",
+  "ManufacturersModelName":"NIRScout",
+  "TaskDescription":"visual gratings and noise patterns",
+  "Instructions":"look at the dot in the center of the screen and press the button when it changes color",
+  "SamplingFrequency":3.7,
+  "NIRSChannelCount":56,
+  "NIRSSourceOptodeCount":16,
+  "NIRSDetectorOptodeCount":16,
+  "ACCELChannelCount":0,
+  "SoftwareFilters":"n/a",
+  "RecordingDuration":233.639,
+  "DCOffsetCorrection":0,
+  "HardwareFilters":{"Highpass RC filter": {"Half amplitude cutoff (Hz)": 0.0159, "Roll-off": "6dBOctave"}},
+  "CapManafacturer":"NIRx",
+  "CapManufacturersModelName":"Headband with print (S-M)",
+}
+```
+
+### Participants file
+
+For fNIRS data the modality agnostic participant.tsv and json files SHOULD contain
+the age field, this is required for calculation of age specific path length factors.
+If the dataset contains multiple groups, for example patients and controls, then
+a column labeled `group` MAY be included (see [participants.tsv](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file)).
+
+```text
+participant_id	age	 sex	hand  group
+sub-01       	34	 M	    n/a   patient
+sub-02	        32	 F	    n/a   control
+```
+
+## Channels description (`*_channels.tsv`)
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["channels"]) }}
+
+Channels are a pairing of source and detector optodes with a specific light wavelength.
+Unlike in other modalities, not all pairings of optodes correspond to meaningful data
+and not all pairs have to be recorded or represented in the data.  Note that the source
+and detector names used in the channel specifications are specified in the `*_optodes.tsv`
+file below. The order of the required columns in the `*_channels.tsv` file MUST be listed as below.
```

Comment:
Do you mean that the required columns must be listed _in the same order_ as below?